test/blockchaintest/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# evmone: Fast Ethereum Virtual Machine implementation
+# Copyright 2022 The evmone Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+hunter_add_package(nlohmann_json)
+find_package(nlohmann_json CONFIG REQUIRED)
+
+add_library(evmone-blockchaintestutils STATIC)
```

Comment:
It does not make sense to create these library. Better place all these in the already existing `statetestutils`. You can keep file locations as they are for now.